login.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>spacTrend</title>
+</head>
+<body>
+    <div class="navbar-section flex-row">
+        <div class="navbar-brand">
+            <span>spacTrend</span> 
+        </div>
+        <div class="search flex-row">
+            <input type="text" class="search-input" placeholder="search for Product, Brand and more">
+            <button class="navbar-icon-btn search-btn"><i class="fas fa-search"></i></button>
+        </div>
+        <div class="navbar-actions flex-row">
+            <button class="navbar-icon-btn"><i class="fas fa-bell"></i></button>
+            <button class="navbar-icon-btn"><i class="far fa-heart"></i></button>
+            <button class="navbar-icon-btn"><i class="fas fa-shopping-cart"></i></button>
+        </div>
+    </div>
+    <div class="login-section flex-row">
+        <div class="login-container flex-column">
+            <h1>Login</h1>
+            <div class="login-credentials-container flex-column ">
+                <div class="login-credential-field flex-column">
+                    <label for="email">Email address</label>
+                    <input type="text" placeholder="abc@gmail.com">
+                </div>
+                <div class="login-credential-field flex-column">
+                    <label for="password">Password</label>
+                    <input type="text" placeholder="**********">
+                </div>
+            </div>
+            <div class="checkbox-link-input flex-row">
+                <div class="checkbox-input">
+                    <input type="checkbox" id="term&condition">
+                    <label for="term&condition">Remember me</label>
+                </div>
+                <a href="">Forgot your password?</a>
+            </div>
+            <button class="btn text-button">Login</button>
+            <button class="btn login-link-button"><a href="">Create New Account</a></button>
```

Comment:
Don't leave href attribute blank like this...... it should be given "#" if not assigned to a link

Reply:
It is supposed to have the signup link. Thank you.

Reply:
Oh....then its fine👍🏻